Guides/dev-set-up-environment.md

```
@@ -8,255 +8,433 @@
     ```
 
-4. Configure Environment Variables:
+    Maybe a couple of notes on branch naming:
+     - We usually try to preface the branch with some kind of reference that relates to the organization we are representing. For example, our Tazama staff will create branches called "tazama/my-branch-name-here". We encourage you to create your branch as "your-org/your-branch-name-here".
+     - The branch name itself should be short, but descriptive, and related to the change that you want to make. "But," you ask: "how do I know what the change is that I'm going to make this soon after cloning the repository?" This is a good question, and this brings us to one of our guiding principles for contributions, and development contribution in particular: your contribution should be intentional and deliberate. You should already know what you are intending to do when you want to modify Tazama code, and, even more to the point, you should have declared your intention in a well-documented GitHub issue in the same repository that you are modifying. We'll get to that a little later in your journey.
+     - Avoid spaces in the naming and rather use dashes or underscores to separate words in your branch name.
 
-    Each microservice processor's configuration is specified as environment variables that are located in a dot env (`.env`) file. Your cloned repository does not have one yet: we'll have to create it from the `.env.template` file that is already in your folder. You can copy this file with from your Windows Command Prompt:
+4. Make a clean start
+
+    This step isn't always necessary, but it's generally good practice.Using `npm`, you can clean up your local repository to make sure there are no lingering build artefacts:
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)